gr-analog/lib/cpm.cc

```diff
@@ -183,8 +180,8 @@ namespace gr {
       double alpha = 5.336446256636997 * bt;
       for(unsigned i = 0; i < samples_per_sym * L; i++) {
 	double k =  i - Ls/2; // Causal to acausal
-	taps_d[i] = (bm::erf(alpha * (k / samples_per_sym + 0.5)) -
-		     bm::erf(alpha * (k / samples_per_sym - 0.5)))
+        taps_d[i] = (erf(alpha * (k / samples_per_sym + 0.5)) -
```

Comment:
Shouldn't this be `std::erf()`?

Reply:
First, it does compile with GCC5.4 without and with the `std` namespace.
Second, http://www.cplusplus.com/reference/cmath/erfc/ claims `'erf` is correct.
Third, http://en.cppreference.com/w/cpp/numeric/math/erf claims `std::erf` is correct.

All in all, this is really tricky. I'd go for the `erf` version since it works. If it breaks on other systems, we need to investigate more.

Reply:
Cplusplus.com is known for misleading examples. The example on that page is C99, not C++11. It probably only works because we're including `math.h` or `tgmath` somewhere else. We really ought to go directly for `std::erf`– it's the overloaded, type safe, C++ way of doing this :)